contracts/acl/ACL.sol

```diff
@@ -156,6 +157,18 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
         _setPermissionManager(address(0), _app, _role);
     }
 
+    /**
+    * @notice Lock `_role` in `_app`, so no midification can be made to it (grant, revoke, permission manager)
+    * @param _app Address of the app in which the permission is being unmanaged
+    * @param _role Identifier for the group of actions being unmanaged
+    */
+    function petrifyPermission(address _app, bytes32 _role)
```

Comment:
Thinking about the name, I think `petrify` could be confused with the aragonOS concept of petrifying apps. I think `lock` could be a better name, it is what you actually used to explain it in the `@notice`.